osu.Game/Overlays/Settings/SettingsFooter.cs

```diff
@@ -67,6 +71,17 @@ private void load(OsuGameBase game, OsuColour colours, RulesetStore rulesets)
                     Colour = DebugUtils.IsDebug ? colours.Red : Color4.White,
                 },
             };
+
+            this.game = game;
+            this.changelog = changelog;
+        }
+
+        protected override bool OnClick(ClickEvent e)
+        {
+            if (!game.IsDeployedBuild) return base.OnClick(e);
```

Comment:
Please invert this `if` so that you only have one call to `base.OnClick`.

Reply:
should return true on handling anyways.